puppet/modules/slave/manifests/packaging/rpm.pp

```diff
@@ -7,6 +7,7 @@
   Optional[String] $koji_certificate = undef,
 ) {
   $is_el7 = $facts['os']['release']['major'] == '7'
+  $ansible_python_version = if $facts['os']['release']['major'] == '8' { 'python3.11' } else { 'python3' }
```

Comment:
I took the liberty to also make this el9 compatible where ansible (again) uses default python.